### ISSUE TITLE
Matplotlib to HoloViews tutorial

### ISSUE THREAD

**Issue by jlstevens** [holoviz/holoviews#132]
It might be nice to have a notebook showing how to do common things people are familiar with in matplotlib in HoloViews. I don't think we necessarily require drawn out examples - just a table of matplotlib commands on the left and HoloViews equivalents on the right would be a good start.

**Comment by jbednar** [comment]
Yes, that would be great.  I bet @basnijholt has already got a list of those in his head, having recently migrated his code from matplotlib to HoloViews; maybe we could convince him to make a first draft?  :-)

**Comment by thoth291** [comment]
I recently came across this website: https://python-graph-gallery.com/
And keeping in mind these educational ideas: https://datavizcatalogue.com/ and your excellent understanding of the plotting pitfalls and rich API of holoviews - you might came up with quite a unique resource which will be used not only to teach people how to plot data but also why to plot it this way and not another.
Currently I have to use both of these websites together with your reference gallery to educate my users.
It would be much more convenient to have it all in one resource...

**Comment by jbednar** [comment]
The Datavizcatalogue is nice, but:

![image](https://user-images.githubusercontent.com/1695496/31734978-042820ba-b406-11e7-8046-5ec39c2c6c0d.png)

I.e., they want payment to include links, and not just payment for the trouble of adding them, but ongoing payment, which amounts to purchasing advertising.  Our projects are all open source, and so advertising doesn't make much sense, given that there is no revenue stream to pay back the ongoing cost of the advertising. 

python-graph-gallery is more likely to be viable as an overview, but all the current tools there are matplotlib-based, so I'm not sure if they would be interested in our more interactive plots.  Certainly something to consider!
